### DISCUSSION TITLE
Blog description body not showing up

### DISCUSSION THREAD

**Discussion by palomachiara** [tinacms/tinacms#3927] [Help] [answered]
Hello
This is what it looks like in my config file:
`      {
        format: "md",
        label: "Posts",
        name: "postsEn",
        path: "content/english/blog",
        frontmatterFormat: "yaml",
        match: {
          include: "**/*",
          exclude: "_index",
        },
        fields: [
          ...blog_postFields(),
        ],
      },`

Inside of blog_postFields, I have:
`    {
      type: "rich-text",
      name: "description",
      label: "Description",
      isBody: true,
    },`

I have text in my blogs, but they're not showing up. Example:
https://kaliriu.com/blog/how-can-web-design-improve-online-sales/
https://ibb.co/HT68jF0

The rest of my code is here:
https://github.com/palomachiara/kaliriu

Is there something I haven't set up correctly?
Thanks in advance

**Comment by logan-anderson** [comment]
@palomachiara  setup looks correct to me. To me, this looks like an issue with your static site generated and not with Tina. It looks like this was already [question was already asked in discord](https://discord.com/channels/835168149439643678/1105820512724189306) and the same repo was linked. From the message, it looks like you where able to sort out what was going on here?

Is there anything else we can do to help?

**Reply by palomachiara** [reply]
I managed to change the Type to rich-text, which was what I asked on discord, but then when I added text I realised it isn't showing up.. It worked fine before I had Tina setup so I don't think the issue is elsewhere.

**Reply by palomachiara** [reply]
It's a Tina issue, as said here: https://discourse.gohugo.io/t/blog-description-body-not-showing-up/44313

**Comment by jeffsee55** [comment, marked as answer]
One issue I see from the post in the Hugo discourse is that you're setting the `description` field to `isBody`. This will result in the `description` field being removed from the frontmatter, and being put in the "body" of the `md` file.

I can no longer see your repo but you might want to try something like this:

```js
{
  format: "md",
  label: "Posts",
  name: "postsEn",
  path: "content/english/blog",
  frontmatterFormat: "yaml",
  match: { include: "**/*", exclude: "_index" },
  fields: [
    {
      type: "string",
      name: "title",
      label: "Title",
    },
    {
      type: "string",
      name: "slug",
      label: "Slug",
    },
    {
      type: "date",
      name: "date",
      label: "Date",
    },
    {
      type: "rich-text",
      name: "meta_description",
      label: "Meta Description",
    },
    {
      type: "rich-text",
      name: "description",
      label: "Description",
    },
    {
      type: "rich-text",
      name: "body",
      label: "Body",
      isBody: true,
    },
  ],
};
```

**Reply by palomachiara** [reply]
Thank you. I already switched to another CMS though so I can't test it anymore.
